### DISCUSSION TITLE
Playtime theme for Reaper itself as a whole to match the vibe and design and to make it look good as a whole

### DISCUSSION THREAD

**Discussion by joungle123** [helgoboss/helgobox#1255] [Ideas]
By the way, fantastic dark theme!! – it would be nice if you could create such a theme for a Reaper itself, I would fucking buy it seriously, cause it looks super good and would like to have it in such a slick modern solid view the whole Reaper that would blend together with Playtime 2 and make the whole creative experience just awesome!!

**Comment by helgoboss** [comment]
Hi! I'm not into REAPER themes making, but maybe someone else out there is interested in doing that?

The only thing I could imagine doing at some point is to write a sort of color palette exporter that you could then use to adjust the REAPER default theme colors.

**Reply by joungle123** [reply]
@helgoboss possibly I will make it myself and share it in the future. As I see that Reaper really lacks good design...
